modules/cluster-node-tuning-operator-verify-profiles.adoc

```diff
@@ -28,3 +28,22 @@ worker-b         openshift-node            True      False      6h28m
 * `APPLIED`: `True` if the TuneD daemon applied the desired profile. (`True/False/Unknown`).
 * `DEGRADED`: `True` if any errors were reported during application of the TuneD profile (`True/False/Unknown`).
 * `AGE`: Time elapsed since the creation of Profile object.
+
+The ClusterOperator/node-tuning object also contains useful information about the Operator and its node agents' health.
+For example, operator misconfiguration will be reported via ClusterOperator/node-tuning status messages.
```

Comment:
There's a couple places where you've written "operator" and "Operator". Guidelines require that it be written capitalized Operator throughout.

Reply:
Ack.  And removed a hard-wrap here too.